src/components/notification/NotificationBadge.tsx

```diff
@@ -0,0 +1,78 @@
+import { Intent, Popover, PopoverInteractionKind, Position, Tag } from '@blueprintjs/core';
+import * as React from 'react';
+import { AcademyNotification, AcademyNotificationType } from './notificationShape';
+
+type OwnProps = {
+  className?: string;
+  enableHover?: boolean; // enable or disable hover popover option
+  large?: boolean; // enable to use large style
+  notifications: AcademyNotification[];
+};
+
+export type DispatchProps = {
+  handleAcknowledgeNotification: (notificationIds: number[]) => void;
+};
+
+const NotificationBadge: React.SFC<OwnProps & DispatchProps> = props => {
+  if (!props.notifications.length) {
+    return null;
+  }
+
+  const makeNotificationTags = (notifications: AcademyNotification[]) => (
+    <div className="col">{notifications.map(makeNotificationTag)}</div>
+  );
+
+  const makeNotificationTag = (notification: AcademyNotification) => {
+    const onRemove = () => props.handleAcknowledgeNotification([notification.id]);
+
+    return (
+      <Tag
+        className="row"
+        style={{ backgroundColor: 'transparent', marginLeft: '0.4rem', marginRight: '0.4rem' }}
```

Comment:
Why is style here instead of in a stylesheet like the other styles?

Reply:
Moved to stylesheet